### PR TITLE
AudiobridgePlugin.connectRoom

### PR DESCRIPTION
**Pull request by vogdb**

### CODE REVIEW

**Comment by vogdb** [comment]
@zazabe please review

File: src/webrtc/plugin/audiobridge-plugin.js [hunk 1]
```diff
@@ -101,7 +116,26 @@ AudiobridgePlugin.prototype.changeRoom = function(roomId, options) {
...
+ * @param {int} roomId
+ * @param {Object} [options]
+ * @param {int} [options.id]
+ * @param {string} [options.display]
+ * @param {boolean} [options.muted]
+ * @param {int} [options.quality]
+ * @return {Promise}
+ */
+AudiobridgePlugin.prototype.connectRoom = function(roomId, options) {
+  if (this._currentRoomId) {
```
Comment (zazabe) [inline src/webrtc/plugin/audiobridge-plugin.js]: should we check if `roomId == this._currentRoomId` and resolve directly in this case?

Comment (vogdb) [inline src/webrtc/plugin/audiobridge-plugin.js]: good catch

**Comment by vogdb** [comment]
@zazabe please review

**Comment by zazabe** [comment]
lgtm

**Pull request merged**